<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Supplier</class>
 <widget class="QWidget" name="Supplier">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>738</width>
    <height>508</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LMS</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/iconwindow.png</normaloff>:/iconwindow.png</iconset>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="4,5">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Supplied_books</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QTableView" name="tableView_supplied_books">
          <property name="selectionMode">
           <enum>QAbstractItemView::SingleSelection</enum>
          </property>
          <property name="selectionBehavior">
           <enum>QAbstractItemView::SelectRows</enum>
          </property>
          <property name="cornerButtonEnabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_supply_book">
            <property name="toolTip">
             <string>Supply a new book to the library</string>
            </property>
            <property name="text">
             <string>Supply book</string>
            </property>
            <property name="icon">
             <iconset resource="../src.qrc">
              <normaloff>:/images/addbook.png</normaloff>:/images/addbook.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_modify_supply">
            <property name="toolTip">
             <string>Modify information for selected supplied book</string>
            </property>
            <property name="text">
             <string>Modify supply</string>
            </property>
            <property name="icon">
             <iconset resource="../src.qrc">
              <normaloff>:/images/modify.png</normaloff>:/images/modify.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_remove_supply">
            <property name="toolTip">
             <string>Cancel selected book supply</string>
            </property>
            <property name="text">
             <string>Remove supply</string>
            </property>
            <property name="icon">
             <iconset resource="../src.qrc">
              <normaloff>:/images/deletebook-icon.png</normaloff>:/images/deletebook-icon.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_refresh">
            <property name="toolTip">
             <string>Refresh</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../src.qrc">
              <normaloff>:/images/refresh.png</normaloff>:/images/refresh.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>20</width>
              <height>20</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Search for books</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,1,0">
          <item>
           <widget class="QComboBox" name="comboBox">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Search by</string>
            </property>
            <item>
             <property name="text">
              <string>All books</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Title</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Author</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Publisher</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Year</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>ISBN</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEdit_search_book">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_search_book">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="toolTip">
             <string>Search</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset resource="../src.qrc">
              <normaloff>:/images/search-icon.png</normaloff>:/images/search-icon.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>24</width>
              <height>24</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableView" name="tableView_books"/>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../src.qrc"/>
 </resources>
 <connections/>
</ui>
